src/test/java/spoon/test/refactoring/testclasses/VariableRename.java

```diff
@@ -0,0 +1,244 @@
+package spoon.test.refactoring.testclasses;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+import static org.junit.Assert.*;
+
+public class VariableRename
+{
+	public VariableRename() throws Throwable
+	{
+		int local1 = 0;
+		//call all not private methods of this class automatically, to check assertions

```

Comment:
interesting. would be event better in a separate clear method. 
new RenameTestSubject().callAllMethods()